From: Andreas Barth <aba@not.so.argh.org>
Date: Sun, 2 Jun 2013 10:51:30 +0200
Subject: adjust several defaults

(cherry picked from commit e7f131cec869c097e9e5f92615b0a3d328cc1bc2)

diff --git a/lib/Buildd/Conf.pm b/lib/Buildd/Conf.pm
index 312614a..df97ecd 100644
--- a/lib/Buildd/Conf.pm
+++ b/lib/Buildd/Conf.pm
@@ -100,7 +100,7 @@ sub setup ($) {
 	    TYPE => 'STRING',
 	    VARNAME => 'admin_mail',
 	    GROUP => 'Mail',
-	    DEFAULT => 'root',
+	    DEFAULT => 'buildd-porters',
 	    HELP => 'email address for admin'
 	},
 	'APT_GET'				=> {
@@ -171,7 +171,7 @@ sub setup ($) {
 	    TYPE => 'BOOL',
 	    VARNAME => 'log_queued_messages',
 	    GROUP => 'Mail',
-	    DEFAULT => 0,
+	    DEFAULT => 1,
 	    HELP => 'Log success messages from upload queue daemon?'
 	},
 	'MAX_SBUILD_FAILS'				=> {
@@ -192,7 +192,7 @@ sub setup ($) {
 	    TYPE => 'NUMERIC',
 	    VARNAME => 'nice_level',
 	    GROUP => 'Build options',
-	    DEFAULT => 10,
+	    DEFAULT => 0,
 	    HELP => 'Nice level to run sbuild.  Dedicated build daemons should not be niced.'
 	},
 	'NO_DETACH'				=> {
@@ -229,14 +229,14 @@ sub setup ($) {
 	    TYPE => 'BOOL',
 	    VARNAME => 'should_build_msgs',
 	    GROUP => 'Daemon',
-	    DEFAULT => 1,
+	    DEFAULT => 0,
 	    HELP => 'Should buildd send "Should I build" messages?'
 	},
 	'STATISTICS_MAIL'			=> {
 	    TYPE => 'STRING',
 	    VARNAME => 'statistics_mail',
 	    GROUP => 'Watcher',
-	    DEFAULT => 'root',
+	    DEFAULT => 'buildd-porters',
 	    HELP => 'email address for statistics summaries'
 	},
 	'STATISTICS_PERIOD'			=> {
diff --git a/lib/Buildd/DistConf.pm b/lib/Buildd/DistConf.pm
index 6edffba..94803bb 100644
--- a/lib/Buildd/DistConf.pm
+++ b/lib/Buildd/DistConf.pm
@@ -98,7 +98,7 @@ sub setup ($) {
 	    DEFAULT => undef,
 	},
 	'WANNA_BUILD_DB_USER'			=> {
-	    DEFAULT => $Buildd::username
+	    DEFAULT => undef,
 	},
 	'WANNA_BUILD_API'			=> {
 	    DEFAULT => undef,
-- 
2.1.4

