From: Andreas Barth <aba@not.so.argh.org>
Date: Tue, 21 May 2013 20:33:34 +0200
Subject: add checks for buildd and auto-create apt gpg keys

(cherry picked from commit da9dbeb6eb13463c0f2eee48640c3a2a0b1af4d1)

diff --git a/bin/buildd b/bin/buildd
index 41fbf09..faa393e 100755
--- a/bin/buildd
+++ b/bin/buildd
@@ -32,6 +32,14 @@ sub shutdown_fast ($);
 sub reread_config ($);
 sub reopen_log ($);
 
+(grep { getgrgid($_) eq "sbuild" } split(" ", $( )) || die ("user needs to be in group sbuild!\n");
+if (! -e "/var/lib/sbuild/apt-keys/sbuild-key.pub" ) {
+    print "need to create sbuild key - this could take its time\n";
+    system("sbuild-update --keygen");
+}
+system("grep -qs ^/var/lib/sbuild/build /etc/schroot/buildd/fstab");
+($? >> 8) || die("should get rid of /var/lib/sbuild/build in /etc/schroot/buildd/fstab (at least for d.o-systems)");
+
 my $conf = Buildd::Conf::new();
 exit 1 if !defined($conf);
 my $options = Sbuild::OptionsBase->new($conf, "buildd", "1");
-- 
2.1.4

