From: Andreas Barth <aba@not.so.argh.org>
Date: Sun, 19 May 2013 22:56:06 +0200
Subject: ignore obsolete max_build in .builddrc (instead of failing)

(cherry picked from commit 8448d1e17f0553a385f5b379e4d01d0e17e5fadb)

diff --git a/lib/Buildd/Conf.pm b/lib/Buildd/Conf.pm
index 0fc3c50..4bdb42a 100644
--- a/lib/Buildd/Conf.pm
+++ b/lib/Buildd/Conf.pm
@@ -408,6 +408,9 @@ undef \@distributions;
 my \@upload_queues;
 undef \@upload_queues;
 
+# unused now but needed to avoid errors with old config during startup
+my \$max_build = undef;
+
 #legacy fields:
 my \@weak_no_auto_build;
 undef \@weak_no_auto_build;
-- 
2.1.4

